bin/Execute_report.R

```diff
@@ -61,8 +64,9 @@ rmarkdown::render(opt$report, output_file = opt$output, knit_root_dir = wd, outp
                                 path_relevel = opt$relevel,
                                 batch_effect = opt$batch_effect,
                                 log_FC_threshold = opt$log_FC_threshold,
-                                nsub_genes = opt$nsub_genes,
                                 rlog = opt$rlog,
+                                nsub_genes = opt$nsub_genes,
```

Comment:
In the `RNAseq_report.Rmd` you changed the `opt` to `params`. It would be nice if you could use it here as well to be consistent.

Reply:
That isn't something that I changed just so; `.R` uses `opt` and `.Rmd` uses `params`